include/fastdds/rtps/transport/TransportInterface.h

```diff
@@ -24,9 +24,9 @@
 #include <fastdds/rtps/transport/TransportReceiverInterface.h>
 #include <fastdds/rtps/network/SenderResource.h>
 
-namespace eprosima{
-namespace fastdds{
-namespace rtps{
+namespace eprosima {
+namespace fastdds {
+namespace rtps {
 
 static const uint32_t s_maximumMessageSize = 65500;
```

Comment:
Should we make these constants be `constexpr` instead?
Should we have documentation for them?

Forward declaration `ChannelResource` is not used

Reply:
Done all suggested changes in 4fcfb6feaa85204124ab730ef1a58d1ee5337c57